libraries/apollo-tooling/src/test/kotlin/com/apollographql/apollo3/tooling/SchemaDownloaderTests.kt

```diff
@@ -9,21 +9,893 @@ import org.junit.Assert.assertTrue
 import org.junit.Test
 import java.io.File
 
-class SchemaDownloaderTests {
-  private lateinit var mockServer: MockServer
-  private lateinit var tempFile: File
+private const val metaSchemaJune2018Response = """
```

Comment:
Nit: can this be moved to separate test fixture files so that we have less scrolling to do?